### ISSUE TITLE
 App Store suddenly not showing any updates even when unjailbroken 

### ISSUE THREAD

**Issue by eglacias** [pwn20wndstuff/Undecimus#2093]
App Store Won’t show or refresh to show updates yet if I click on a individual app via search it will show that app itself needs an update.

 iPhone 6s Plus, iOS 12.4.1, uncover 5.2.

This just showed up today randomly, haven’t had a problem in months. I’ve tried every listed solution on the Internet, nothing does a thing. Delete AppStore plist, icleaner, etc. 

 This is a common problem that seems somehow related to jailbreaking. It has never been solved or properly addressed. I would debug, copy files, or anything to help figure this out in a more systemic and concrete fashion!

Obviously this is an absolute showstopper when it occurs, it is no minor issue just to put aside.

**Comment by eglacias** [comment]
Errr...  I have never ever had to pull to refresh in the App Store. I tried and The updates went from blank to all listed when I did that on Wi-Fi just now.

So, likely the following procedure works, but you have to go back to updates and pull down to refresh to get the files to re-populate: 

 TL;DR Kill the App Store if open, go to /var/mobile/Library/Preferences using Filza or SSH, delete com.apple.AppStore.plist and com.apple.appstored.plist then do a soft reboot (or a full reboot + rejailbreak).

But first try just to enable password protection for free items (in Settings -> iTunes Store and App store -> Password Settings)

**Comment by eglacias** [comment]
This being a common issue, I see nowhere but assume they’re out to be a troubleshooting section with these solutions. Is there such a thing so like this can be added to?
